disco_aws_automation/disco_vpc.py

```diff
@@ -750,68 +750,119 @@ def find_vpc_id_by_name(vpc_name):
             raise MultipleVPCsForVPCNameError("More than 1 VPC is named as {}".format(vpc_name))
 
     @staticmethod
-    def parse_peering_connection_line(line, vpc_conn):
+    # Pylint thinks this function has too many local variables
+    # pylint: disable=R0914
+    def parse_peering_connection_line(line, existing_vpcs):
```

Comment:
This function is getting really big, probably needs to be broken down a little. One possibility is to make it just produce all the peering pairs and let the caller do the mapping to the existing VPCs. I could do that in the VPC update story too.